runtime-modules/content/src/tests/migration.rs

```diff
@@ -57,97 +146,15 @@ fn migration_test() {
-        );
-        assert_err!(
-            Content::create_channel(
-                Origin::signed(FIRST_MEMBER_ORIGIN),
-                ContentActor::Member(FIRST_MEMBER_ID),
-                ChannelCreationParametersRecord {
-                    assets: None,
-                    meta: Some(vec![]),
-                    reward_account: None,
-                },
-            ),
-            Error::<Test>::MigrationNotFinished
-        );
-        assert_err!(
-            Content::update_channel(
-                Origin::signed(FIRST_MEMBER_ORIGIN),
-                ContentActor::Member(FIRST_MEMBER_ID),
-                <Test as storage::Trait>::ChannelId::one(),
-                ChannelUpdateParametersRecord {
-                    assets_to_upload: None,
-                    new_meta: Some(vec![]),
-                    reward_account: None,
-                    assets_to_remove: BTreeSet::new(),
-                },
-            ),
-            Error::<Test>::MigrationNotFinished
-        );
-        assert_err!(
-            Content::update_video(
-                Origin::signed(FIRST_MEMBER_ORIGIN),
-                ContentActor::Member(FIRST_MEMBER_ID),
-                <Test as Trait>::VideoId::one(),
-                VideoUpdateParametersRecord {
-                    assets_to_upload: None,
-                    new_meta: Some(vec![]),
-                    assets_to_remove: BTreeSet::new(),
-                },
-            ),
-            Error::<Test>::MigrationNotFinished
-        );
-        assert_err!(
-            Content::delete_video(
-                Origin::signed(FIRST_MEMBER_ORIGIN),
-                ContentActor::Member(FIRST_MEMBER_ID),
-                <Test as Trait>::VideoId::one(),
-                BTreeSet::new(),
-            ),
-            Error::<Test>::MigrationNotFinished
-        );
-        assert_err!(
-            Content::delete_channel(
-                Origin::signed(FIRST_MEMBER_ORIGIN),
-                ContentActor::Member(FIRST_MEMBER_ID),
-                <Test as storage::Trait>::ChannelId::one(),
-                0u64,
-            ),
-            Error::<Test>::MigrationNotFinished
-        );
-        assert_err!(
-            Content::update_video_censorship_status(
-                Origin::signed(FIRST_MEMBER_ORIGIN),
-                ContentActor::Member(FIRST_MEMBER_ID),
-                <Test as Trait>::VideoId::one(),
-                false,
-                b"test".to_vec()
-            ),
-            Error::<Test>::MigrationNotFinished
-        );
-        assert_err!(
-            Content::update_channel_censorship_status(
-                Origin::signed(FIRST_MEMBER_ORIGIN),
-                ContentActor::Member(FIRST_MEMBER_ID),
-                <Test as storage::Trait>::ChannelId::one(),
-                false,
-                b"test".to_vec()
-            ),
-            Error::<Test>::MigrationNotFinished
-        );
+        helper_test_all_relevant_extrinsics();
```

Comment:
Its not clear from the name what you are testing for.

Reply:
Also for completeness it makes sense to test also that after migration is complete that extrinsics succeed.